examples/regtest/integrators/h2o-md_nht/data.lmp

```diff
@@ -0,0 +1,1331 @@
+LAMMPS Description
```

Comment:
Also here, would be great to have a smaller regtest. 216 water molecule seems a bit excessive. What is the idea behind this? I see that it is only for 20 steps that you run the regtest, but still.

Reply:
I think that all regtests for LAMMPS, are based on 216 molecules of water. They are very quick to run. The reason I have this tests is because, NPT dynamics gives different results (which are basically small numerical differences in the 6-8th decimal place) with py3, if a thermostat is used. When the thermostat is switched off, and the NPH ensemble is sampled, the results are the same when comparing across different versions of python. Hence I added this one.

Reply:
ok!